core/cppgc.rs

```diff
@@ -182,10 +182,12 @@ impl FunctionTemplateData {
   }
 }
 
+#[derive(Debug)]
```

Comment:
the debug output is probably quite useless; any specific need for it?

Reply:
The intent is that this would allow `#[derive(Debug)]` to be added to `DOMQuadInner`, but I don't have a particularly strong opinion.

Reply:
oh i see, that makes sense!